src/styles/main.scss

```
@@ -0,0 +1,112 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  height: 100vh;
+  width: 100%;
+
+}
+
+$field_size: 250px;
+$center_dot_size: 10px;
+$number_size: 40px;
+$pointer_size: 100px;
+$width_pointer: 3px;
+$number_font-size: 30px;
+$number_font-family: "Arial, Helvetica, sans-serif";
+
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: $field_size;
+  height: $field_size;
+  border: 4px solid #091921;
+  border-radius: 50%;
+  box-shadow: 0 -15px 15px rgba(255, 255, 255, 0.05),
+    inset 0 -15px 15px rgba(255, 255, 255, 0.05),
+    0 15px 15px rgba(0, 0, 0, 0.3),
+    inset 0 15px 15px rgba(0, 0, 0, 0.3);
+  background-image: url(../pictures/sec.png);
+  object-position: top left;
+  object-fit: cover;
+
+  &__dot {
+    position: absolute;
+    width: $center_dot_size;
+    height: $center_dot_size;
+    border: 2px solid #091921;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+    background-color: black;
+    z-index: 999;
+  }
+
+  &__min {
+    position: absolute;
+
+    .minutes {
+      display: flex;
+      justify-content: center;
+      position: relative;
+      width: 130px;
+      height: 130px;
+      border-radius: 50%;
+      animation: minutes 720s linear infinite;
```

```suggestion
      animation: minutes 3600s linear infinite;
```
why 720?
now after 1 turn of second arrow it will pass near 5 minutes...